rnote-ui/src/main.rs

```diff
@@ -77,3 +82,32 @@ fn setup_windows_env() -> anyhow::Result<()> {
     //std::env::set_var("RUST_LOG", "rnote=debug");
     Ok(())
 }
+
+/// we need to set some env vars for macos app bundles
+#[cfg(target_os = "macos")]
+fn setup_macos_env() -> anyhow::Result<()> {
+    use std::ffi::OsStr;
+    use std::path::{Component, PathBuf};
+
+    let current_dir = std::env::current_dir()?.canonicalize()?;
+    if current_dir
+        .components()
+        .zip(current_dir.components().skip(1))
+        .any(|(a, b)| {
+            if let (Component::Normal(a), Component::Normal(b)) = (a, b) {
+                a == OsStr::new("Contents") && b == OsStr::new("MacOS")
+            } else {
+                false
+            }
+        })
+    {
+        std::env::set_var("XDG_DATA_DIRS", &current_dir.join("/../Resources/share"));
```

Comment:
Apologies if my assumption is incorrect: I think `current_dir` does not contain the directory name but still the full path to the binary at this point. I'd expect a `skip()` here same as above.

Reply:
no, you were right.